src/context/NameContext.js

```diff
@@ -0,0 +1,30 @@
+import React, { createContext, useState, useEffect } from 'react';
+// import PropTypes from 'prop-types';
+
+export const NameContext = createContext();
+
+const NameContextProvider = (props) => {
```

Comment:
I think you can just do `({ children  })` here and avoid the disable lint below?

Reply:
I tried to change this and the linter still found an error. So I just reverted to the original and will look at solutions that don't require disabling the linter.